helm/templates/configmap.yaml

```diff
@@ -1,6 +1,7 @@
 {{- $chartName := include "gpkg-merger.name" . -}}
 {{- $tracingUrl := include "gpkg-merger.tracingUrl" . -}}
 {{- $metricsUrl := include "gpkg-merger.metricsUrl" . -}}
+{{- $httpProtocol := ternary "https://" "http://" .Values.rasterCommon.ca.secretName -}}
```

Comment:
Create and use instead `.Values.rasterCommon.storage.s3.ssl.enabled` or something like that.
This should be specific for S3.

Reply:
fixed

Reply:
fixed